docs/backends/ldap.rst

```diff
@@ -25,7 +25,7 @@ supported.
 
 The original author for this module is Norbert Sendetzky. This page is
 based on the content from his `LDAPbackend wiki
-section <http://wiki.linuxnetworks.de/index.php/PowerDNS_ldapbackend>`__
+section <https://web.archive.org/web/20160212235114/http://www.linuxnetworks.de/doc/index.php/PowerDNS_LDAP_Backend>`__
```

Comment:
For a number of `http:` sites where the site no longer has the content, I'm replacing links with web.archive.org references.